docs/architecture/cloud-native/logging-with-elastic-stack.md

```diff
@@ -76,7 +77,7 @@ A query that searches for the top 10 pages visited by `jill@example.com`, appear
   }
 ```
 
-**Figure 7-4** - An Elasticsearch query for finding top 10 pages visited by a user
+**Figure 7-7** - An Elasticsearch query for finding top 10 pages visited by a user
 
 ## Visualizing information with Kibana web dashboards
```

Comment:
Can you have a screenshot of the Kibana dashboard here?

Reply:
Yes, I need to create one.

Reply:
Actually, using the one you linked from eShopOnContainers.